### DISCUSSION TITLE
RE:  WIll Web3Modal and/or Web3Wallet be compatible with NEAR or ALEO?

### DISCUSSION THREAD

**Discussion by AlexWBroughton** [WalletConnect/walletconnect-monorepo#4199] [Ideas]
RE:  WIll Web3Modal and/or Web3Wallet be compatible with NEAR or ALEO?  Is there MPC integration for semi-custody of keys?

**Comment by glitch-txs** [comment]
Hi Web3Wallet is chain agnostic, as well as universal-provider from the dapp side
